cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -47,6 +48,7 @@
 
     # cgroup: devices
     capability sys_admin,
+    capability dac_read_search,
```

Comment:
Can you explain why you needed this change please?

Reply:
That's the capability which snap was asking for, see my commit message. It can be related to recent kernel change which affected selinux: https://danwalsh.livejournal.com/77140.html .

Maybe capability dac_override isn't needed?

Reply:
In older kernels it will ask first for dac_override so dac_read_search wasn't even tried first.

Reply:
Heres's the commit: https://github.com/torvalds/linux/commit/2a4c22426955d4fc04069811997b7390c0fb858e

Reply:
We probably need this in the SELinux policy too...

Reply:
@Conan-Kudo can you try to make that happen please?

Reply:
@zyga When I get some time, yes.

Reply:
@zyga Done: https://github.com/snapcore/snapd/pull/5011